edi_sale_oca/demo/edi_configuration.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="demo_edi_configuration_confirmed" model="edi.configuration">
+        <field name="name">Demo Sale OrderResponse - order confirmed</field>
+        <field
+            name="description"
+        >Show case how you can send out an order response automatically</field>
+        <field name="type_id" ref="demo_edi_exc_type_order_out" />
+        <field name="backend_id" ref="demo_edi_backend" />
+        <field name="model_id" ref="sale.model_sale_order" />
+        <field name="trigger_id" ref="edi_conf_trigger_sale_order_state_change" />
+        <field name="snippet_do">
+# STATES
+# ('draft', 'Quotation'),
+# ('sent', 'Quotation Sent'),
+# ('sale', 'Sales Order'),
+# ('done', 'Locked'),
+# ('cancel', 'Cancelled'),
+if record.state == 'sale':
+  record._edi_send_via_edi(conf.type_id)
```

Comment:
Could be useful to add help text explaining which variables are available to use in the snippet (like Odoo does by default for server actions or crons)

Reply:
variables come from edi_oca. I'll check there.